### PR TITLE
fix(ui-compo): remove duplicated div grid layout

### PR DESCRIPTION
**Pull request by mariocao**
# PR Prelude

Thank you for contributing to sheikah! :)

**Please fill in this template (by putting an `x` inside
the brackets) _before_ filing your PR:**

- [x] I have read and understood [CODE_OF_CONDUCT][code] document.
- [x] I have read and understood [CONTRIBUTING][cont] document.
- [x] I have read and understood [STYLEGUIDE][style] document.
- [x] I have included tests for the changes in my PR. If not, I have included a
  rationale for why I haven't.
- [x] My code is formatted and is accepted by `yarn fmt`.
- [x] **I understand my PR may be closed if it becomes obvious I didn't
  actually perform all of these steps.**

# Why this change is necessary and useful

[Please explain **in detail** why the changes in this PR are needed.]

This PR removes a duplicated grid layout in the data requests section. It also refactors some of the styles so that the smart contracts and data requests sections are using the styles in a similar way.

[code]: https://github.com/witnet/sheikah/blob/master/.github/CODE_OF_CONDUCT.md
[cont]: https://github.com/witnet/sheikah/blob/master/.github/CONTRIBUTING.md
[style]: https://github.com/witnet/sheikah/blob/master/docs/STYLEGUIDE.md

### CODE REVIEW

File: app/renderer/ui/components/main/sections/dataRequests/tabs/editor/index.tsx [hunk 1]
```diff
@@ -33,7 +34,7 @@ class TabEditor extends TabComponent<any & Props> {
   // tslint:disable-next-line:prefer-function-over-method completed-docs
   public render() {
     return (
-      <div className={styles.layout}>
+      <div className={layoutStyles.layout}>
```
Comment (mmartinbar) [inline app/renderer/ui/components/main/sections/dataRequests/tabs/editor/index.tsx]: Mmm, I find it a bit weird that the styles for the data requests tabs are obtained from the smart contracts section. If the layout styles or if the tab styles are common to all tabs, I would perhaps move them to another folder (`tabs-common` or something like that).

Comment (mmartinbar) [reply app/renderer/ui/components/main/sections/dataRequests/tabs/editor/index.tsx]: I think that there may be already an issue related to [this](https://github.com/witnet/sheikah/issues/488), so I'm ok with merging this as is for time purposes and coming back to this issue afterwards.

File: app/renderer/ui/components/main/sections/dataRequests/style.scss [hunk 2]
```diff
@@ -5,4 +5,11 @@
...
     padding: 0;
   }
+
+  .paddedLayout {
+    display: grid;
+    grid-gap: 2em;
+    grid-template-columns: 3fr 1fr;
+    padding: 3em;
+  }
 }
```
Comment (mmartinbar) [inline app/renderer/ui/components/main/sections/dataRequests/style.scss]: I think this is not being used

File: app/renderer/ui/components/main/sections/dataRequests/tabs/history/index.tsx [hunk 3]
```diff
@@ -7,6 +7,7 @@ import { Services } from "app/renderer/services"
 import { CardDefault } from "app/renderer/ui/components/card"
 
 const svgImage = require("svg/dataRequestsHistory.svg")
+const layoutStyles = require("app/renderer/ui/components/main/sections/smartContracts/style.scss")
```
Comment (mmartinbar) [inline app/renderer/ui/components/main/sections/dataRequests/tabs/history/index.tsx]: I think here you may have wanted to import from `dataRequests`, not from `smartContracts`

File: app/renderer/ui/components/main/sections/dataRequests/tabs/editor/index.tsx (hunk 1 above)
Comment (mariocao) [reply app/renderer/ui/components/main/sections/dataRequests/tabs/editor/index.tsx]: Ups sorry...that was a typo should be from the dtaa requests folder...

**Comment by mmartinbar** [comment]
I'm closing the PR as this is already been tackled in #530

**Pull request closed without merging**